### DISCUSSION TITLE
About roles

### DISCUSSION THREAD

**Discussion by TrendyBos** [pkgxdev/pkgx#960] [Help]
Pls how can I get roles in Tea protocol.

**Comment by jetmiky** [comment]
Join the [Discord](https://discord.gg/DG4XNStA).

Currently there are 2 roles available by achieving levels:
- PowerUser: earn over 4,675 XP (Level 10)
- TeaSquad: earn over 101,675 (Level 35)

Ways to get XP are as follows:
20XP: Create content in ⁠memes or ⁠community-content (Post in Discord channel)
20XP: Attend an event in ⁠office-hour or ⁠tea-time (Discord sound channel)
50XP: Retweet/Engage with @teaprotocol on ⁠twitter
100XP: Repost a Discord Q&A on GitHub Discussions.
500XP: Answer a question on GitHub Discussions.
500XP: Create an accepted Pull Request.
2000XP: Build something with tea (release it on Hacker News, Product Hunt or dev.to).

**Reply by rafiki443** [reply]
Sure

**Comment by rafiki443** [comment]
> Pls how can I get roles in Tea protocol.

Is easy mate

**Comment by pet0001** [comment]
> Pls how can I get roles in Tea protocol.

 You have to join on the discord server which you can find on tea's site ( tea.xyz ) and scroll down to the bottom on the page and just press on the discord logo.

 1)There is the @ PowerUser role which you can unlock by earning 4,675XP (Level 10) and also unlocks the power-users channel. 

 2)There is the @ TeaSquad role which you can unlock by earning  101,675XP (Level 35) and also unlocks the teasquad channel. 

 3)There is the @ Moderator role which you can unlock by helping to curate the conversation and also unlocks the moderator channel and 
    give you moderator rights.

 4)There is the @ Contributor  which you can unlock by contributing to tea (through a pull request and you must have your Discord handle in 
    your GitHub profile) and also unlocks the contributors channel. 

 5)There is the @ Team role which you only get by joining the team and also grants you a monthly income


 *Except from these roles there is the basic New Member role which we all get when joining.
